n2y/config.py

```diff
@@ -0,0 +1,43 @@
+import json
```

Comment:
Nice! Pulling this out into it's own file makes total sense since it's likely to grow in size.